### PR TITLE
[Merged by Bors] - chore(SetTheory/Ordinal/Arithmetic): golf two definitions

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Ordinal/Basic.lean [hunk 1]
```diff
@@ -363,9 +363,8 @@ instance NeZero.one : NeZero (1 : Ordinal) :=
 `α.toType` into `β.toType`. -/
 def initialSegToType {α β : Ordinal} (h : α ≤ β) :
     @InitialSeg α.toType β.toType (· < ·) (· < ·) := by
-  change α.out.r ≼i β.out.r
-  rw [← Quotient.out_eq α, ← Quotient.out_eq β] at h; revert h
-  cases Quotient.out α; cases Quotient.out β; exact Classical.choice
+  apply Classical.choice (type_le_iff.1 _)
```
Comment (grunweg) [inline Mathlib/SetTheory/Ordinal/Basic.lean:366]: I personally think using `.mp` or `.mpr` is cleaner than `.1` or `.2`: I will not insist on this, however. Same below.

**Review by grunweg — COMMENTED** [review]
This looks clearly better. Thanks!
maintainer merge

**Review by jcommelin — COMMENTED** [review]
bors d+

**Pull request closed without merging**